### PR TITLE
Snap 1120 fix loading bug

### PR DESCRIPTION
**Pull request by TylerOSI**
### Jira Story

- [Name of Story INT-NNN](https://osi-cwds.atlassian.net/browse/INT-NNN)

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 

The change was in a container method.

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/common/url_helper.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const urlHelper = (path) => {
+  return (
+    `<%= RAILS_RELATIVE_URL_ROOT %>${path}`
```
Comment (TylerOSI) [inline app/javascript/common/url_helper.js]: this is a problem going to figure out how to deal with it now

**Comment by TylerOSI** [comment]
Jenkins test this please

**Comment by pokharelmanish** [comment]
retest this please

**Comment by TylerOSI** [comment]
Jenkins test this please

**Pull request merged**